tools/run_tests/xds_k8s_test_driver/framework/infrastructure/traffic_director.py

```diff
@@ -630,8 +630,7 @@ class TrafficDirectorSecureManager(TrafficDirectorManager):
     netsec: Optional[_NetworkSecurityV1Alpha1]
     SERVER_TLS_POLICY_NAME = "server-tls-policy"
     CLIENT_TLS_POLICY_NAME = "client-tls-policy"
-    # TODO(sergiitk): Rename to ENDPOINT_POLICY_NAME when upgraded to v1beta
```

Comment:
So the original comment said "when upgraded to v1beta". But I notice the name `NetworkServicesV1Alpha1` is not changed. Should that be changed to `NetworkServicesV1Beta` to be correct? May be not in this PR since we have a deadline

Reply:
This is a good point. I noticed this too, and finishing upgrade to v1beta1 in a follow up PR.